mdpow/tests/test_automated_dihedral_analysis.py

```diff
@@ -151,16 +173,22 @@ def test_dihedral_groups_ensemble(self, gen_data):
 
         dh2_mean == pytest.approx(self.DG_C13141520_mean)
         dh2_var == pytest.approx(self.DG_C13141520_var)
-        
-    def test_save_df(self, gen_data, SM25_tmp_dir):
-        dihedrals.save_df(df=gen_data[2], df_save_dir=SM25_tmp_dir, molname='SM25')
-        assert SM25_tmp_dir / 'SM25' / 'SM25_full_df.bz2'
-                                        
 
-    @pytest.mark.skipif(sys.version_info < (3, 8), reason="scipy circvar gives wrong answers") 
-    def test_periodic_angle(self, gen_data):
+    def test_save_df(self, dihedral_data, SM25_tmp_dir):
+        dihedrals.save_df(df=dihedral_data[0], df_save_dir=SM25_tmp_dir, molname='SM25')
+        assert (SM25_tmp_dir / 'SM25' / 'SM25_full_df.csv.bz2').exists(), 'Compressed csv file not saved'
+
+    def test_save_df_info(self, dihedral_data, SM25_tmp_dir, caplog):
+        caplog.clear()
+        caplog.set_level(logging.INFO, logger='mdpow.workflows.dihedrals')
+        dihedrals.save_df(df=dihedral_data[0], df_save_dir=SM25_tmp_dir, molname='SM25')
+        assert f'Results DataFrame saved as {SM25_tmp_dir}/SM25/SM25_full_df.csv.bz2', 'Save location not logged or returned'
```

Comment:
This assertion is always true — it's an non-empty string.

assert something about caplog